frontend/app/src/components/helper/PremiumLock.vue

```diff
@@ -17,8 +18,16 @@
   </v-tooltip>
 </template>
 <script lang="ts">
-import { Vue, Component } from 'vue-property-decorator';
+import { Vue, Component, Prop } from 'vue-property-decorator';
 
 @Component({})
-export default class PremiumLock extends Vue {}
+export default class PremiumLock extends Vue {
+  @Prop({ type: String, required: false, default: '' })
+  size!: string;
+
+  get lockSize() {
```

Comment:
I have an idea of what this might do but a more detailed description would be nice :)

Reply:
Will add.